config/.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+AllCops:
+	Exclude:
+		- 'db/**/*'
+		- 'config/**/*'
```

Comment:
Looking good!

Here are some defaults I recommend, will make it a bit easier on us :)

```
AllCops:
  Exclude:
    - bin/**/*
    - db/**/*
    - vendor/**/*

Rails:
  Enabled: true

# Disables "Use nested module/class definitions instead of compact style" warning
ClassAndModuleChildren:
  Enabled: false

# Increases "Class definition is too long" warning from 100 to 200
Metrics/ClassLength:
  Max: 200

# Increases "Module has too many lines" warning from 100 to 200
Metrics/ModuleLength:
  Max: 200

# Increases "Assignment Branch Condition size" warning from 15 to 30
Metrics/AbcSize:
  Max: 30

# Disables "Missing top-level class documentation comment" warning
Documentation:
  Enabled: false

# Disables "Line is too long" warning
LineLength:
  Enabled: false

# Disables "Method has too many lines" warning
MethodLength:
  Enabled: false
```